acceptancetests/deploy_stack.py

```diff
@@ -520,7 +520,7 @@ def deploy_job():
     return _deploy_job(args, charm_series, series)
 
 
-def update_env(env, new_env_name, series=None, bootstrap_host=None,
+def update_env(env, new_env_name, series=None, arch=None, bootstrap_host=None,
```

Comment:
We don't seem to be using this input.

Reply:
So we don't use it, but it's required. If you try and remove it and deploy it blows up. I don't believe we actually care about the value, just that update_env conforms to a func.